examples/rootca.srl

```diff
@@ -0,0 +1 @@
+5324F08044C2BCB1081F5F527FF04D75AF9C760A
```

Comment:
This also shouldn't be committed.

Reply:
Fixed, and updated gen-certs.sh to follow this policy.